webapp/src/components/cardDetail/cardDetail.tsx

```diff
@@ -116,8 +114,7 @@ const CardDetail = (props: Props): JSX.Element|null => {
     saveTitleRef.current = saveTitle
     const intl = useIntl()
 
-    const clientConfig = useAppSelector<ClientConfig>(getClientConfig)
-    const newBoardsEditor = clientConfig?.featureFlags?.newBoardsEditor || false
```

Comment:
Not sure why this is under feature flags but seems that this is some config setting different than Boards Product feature flag?

Reply:
What I remember is that @jespino was working on new boardsEditor, not sure whether it was public or not.

Reply:
I think for the purpose of this update, we should leave as is

Reply:
Yes, let's not change that for now as it is not in scope and we don't have context yet on that new editor effort